doc/api/tls.md

```diff
@@ -805,6 +856,21 @@ and their processing can be delayed due to packet loss or reordering. However,
 smaller fragments add extra TLS framing bytes and CPU overhead, which may
 decrease overall server throughput.
 
+### tlsSocket.ssl.verifyError()
+<!-- YAML
+added: REPLACEME
```

Comment:
@sam-github @bnoordhuis ... off the top of your heads... do you know what version this was added?